Update plans.mdx (#610)
evcc-io/docs: i18n/en/docusaurus-plugin-content-docs/current/features/plans.mdx

This page needs to be translated from German to English.

> :::

Reviewer: can be removed

> If you enter **a departure time and a loading destination**, the planning algorithm will automatically find the best loading times.

Reviewer: ```suggestion
If you enter **a departure time and a charging goal**, the planning algorithm will automatically find the best loading times.
```
see screenshot

> The planner is only active in PV and Min+PV mode.

Reviewer: ```suggestion
The planner is only active in Solar and Min+Solar mode.
```

Self-produced solar energy continues to be preferred.
If your own energy is not enough, the rest is charged from the power grid.

> Depending on whether you have one [dynamic electricity tariff](./dynamic-prices) or a data source for [CO₂-predictions](./co2) currently have **three loading strategies**:

Reviewer: ```suggestion
Depending on whether you have one [dynamic electricity tariff](./dynamic-prices) or a data source for [CO₂-predictions](./co2) currently have **three charging strategies**:
```

1. **Cheap charging** (dynamic electricity tariff available): Vehicle charges in the cheapest hours before departure.
2. **Green charging** (CO₂ forecast available): Vehicle charges in the hours with the lowest CO₂ emissions.
3. **Standard**: The planner charges the required energy in the hours shortly before departure. This means that the vehicle's battery is still warm when you leave and less active heating energy is required.

The loading strategies cannot be selected manually, but are selected automatically based on the stored data source.

## Create charging plan

To set a plan, click on **Plan** at the charging point.
If evcc knows the charge level of your vehicle (online API) and the battery capacity, you can enter the **desired charge level in %** in addition to the departure time in the dialog.
You will be shown a preview of the charging times.
Use the "Active" switch to activate your plan.

<Screenshot
  name="features/screenshots/plan-soc"

>   caption="Screenshot einer Ladeplanung mittels Ladestand"

Reviewer: translation

/>

If the charge level and capacity are not known, planning is done by specifying an **energy quantity in kWh**.

<Screenshot
  name="features/screenshots/plan-energy"

>   caption="Screenshot einer Ladeplanung mittels Energiemenge"

Reviewer: translation
